lib/ansible/module_utils/network/nxos/config/interfaces/interfaces.py

```diff
@@ -66,11 +78,12 @@ def execute_module(self):
         commands = list()
         warnings = list()
 
-        existing_interfaces_facts = self.get_interfaces_facts()
-        commands.extend(self.set_config(existing_interfaces_facts))
+        existing_interfaces_facts = self.get_interfaces_facts(get_default_interfaces=True)
```

Comment:
Please add note explaining why you only need to set `get_default_interfaces` to `True` when calling from config but not facts.

Reply:
Added a comment to the method header.